test/src/unit-capi-array.cc

```diff
@@ -38,6 +38,7 @@
 
 #include "catch.hpp"
 #include "test/src/helpers.h"
+#include "test/src/vfs_test_helper.h"
```

Comment:
For more similar naming with the existing `helpers.h`, let's name these new files `vfs_helpers.h/cc`